src/Microsoft.Framework.PackageManager/Commands/WrapCommand.cs

```diff
@@ -731,6 +731,33 @@ private static void SetPropertyValueIfEmpty(JObject obj, string name, string val
             }
         }
 
+        private static string ResolveRootDirectory(string projectPath)
```

Comment:
This looks like common helper method. So should this actually be placed in `ProjectResolver`?